main.go

```diff
@@ -14,6 +14,7 @@ type args struct {
 	Config      *string
 	Shell       *string
 	PWD         *string
+	ShowDebug   *bool
```

Comment:
Would rename to `Debug`

Reply:
The thing is Debug is a reserved keyword in powershell I think, so the parameter in powershell and in oh-my-posh will have a different name.

Reply:
That's fine for me.